prow/cluster/jobs/all-periodics.yaml

```diff
@@ -398,6 +398,32 @@ periodics:
     nodeSelector:
       testing: test-pool
 
+# Security policy fuzz testing
+- cron: "0 7 * * *"  # starts every day at 07:00AM UTC
```

Comment:
I think the all-periodics is sort of legacy file. We should probably clean them up long term. For this PR, it would be good to use the new way at least.
An example can be found in the  update-istio-ref job in prow/config/jobs/istio.io.yaml.

Reply:
thanks for the review, updated.